env/env.go

```diff
@@ -81,6 +81,12 @@ const (
 	ScreenshotsOutput = "K6_BROWSER_SCREENSHOTS_OUTPUT"
 )
 
+const (
```

Comment:
To be consistent?

```go
// Infrastructural.
const (
...
```

Something like that.

Reply:
Added in e8987b56d662397bce3d0c4be7be5f991cf075c1